components/metal/FakeUI.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <article class="fake-ui">
     <div class="fake-ui__wrapper">
-      <img src="images/metal/fake-ui.svg" alt="fake software user interface" class="fake-ui__bg">
+      <img src="images/metal/fake-ui.png" alt="fake software user interface" class="fake-ui__bg">
```

Comment:
@korgan00 - I'm sorry I have to add this, but the designs have it so that the entire image changes as the user scrolls down the page. 

Please see the images provided in Figma.

thank you

Reply:
It is now adapted to the new design.